Content.Shared/Blocking/Components/SharedBlockingComponent.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Content.Shared.Actions.ActionTypes;
+using Content.Shared.Sound;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Shared.Blocking;
+
+/// <summary>
+/// This component goes on an item that you want to use to block
+/// </summary>
+[RegisterComponent]
+public sealed class SharedBlockingComponent : Component
+{
+    /// <summary>
+    /// The entity that's blocking
+    /// </summary>
+    [ViewVariables]
+    public EntityUid? User;
+
+    /// <summary>
+    /// Is it currently blocking?
+    /// </summary>
+    [ViewVariables]
+    public bool IsBlocking;
+
+    /// <summary>
+    /// The ID for the fixture that's dynamically created when blocking
+    /// </summary>
+    public string BlockFixtureID = "blocking-active";
```

Comment:
Just make it constant.

Reply:
I can't, I get an error on the system.

![image](https://user-images.githubusercontent.com/54602815/174487794-cf8eef7f-0d54-42f7-a5dc-e1b24d258005.png)

Reply:
Nope, does not allow me to do that. Gives me the same error as above.

Reply:
I just now realized the ? wasn't supposed to be part of it.